roles/scylla-collectd-client/tasks/main.yaml

```diff
@@ -1,5 +1,17 @@
+- name: Gather facts
+  action: ec2_facts
+
+- set_fact: node="scylla-{{ansible_hostname }}"
```

Comment:
oops, already merge before I saw you comment.
From my testing this have no affect.